### ISSUE TITLE
ics/ical script support for multiple calendars

### ISSUE THREAD

**Issue by derekantrican** [derekantrican/GAS-ICS-Sync#6]
From Google-Apps-Script-Library created by [derekantrican](https://github.com/derekantrican) : derekantrican/Google-Apps-Script-Library#10  

The argument here is to add support for multiple ics/ical calendars to be imported into the same calendar

**Comment by agentd00nut** [comment]
See #9

**Comment by p5hema2** [comment]
Simply Copy the Script

**Comment by derekantrican** [comment]
@p5hema2 Making copies of the script is a possible temporary solution, but a PR is already being worked on for a final solution. You can see pull requests related to this issue at #36 and #9 (I'm waiting on approval of #31 to happen first before I start a new PR for this issue, because it will greatly refactor the code)

**Comment by lbell** [comment]
+1 - I see the pull requests in progress, so don't want to muddy the waters by diving in head first. How can I help make this happen?

**Comment by derekantrican** [comment]
@lbell We're working on #47 first and if you feel like you can contribute to that upgrade you can do pulls (or test, open issues, etc) to @jonas0b1011001 's repo here: https://github.com/jonas0b1011001/GAS-ICS-Sync-CalAPIv3

**Comment by derekantrican** [comment]
### Closing as this is fixed by the latest release: [v5.0](https://github.com/derekantrican/GAS-ICS-Sync/releases/tag/5.0)

*If you find that there are still bugs, please comment on this issue and let us know*
